kai/data/templates/main.jinja

```diff
@@ -26,7 +26,7 @@ After you have shared your step by step thinking, provide a full output of the u
 File name: "{{ src_file_name }}"
 Source file contents:
 ```{{ src_file_language }}
-{{ src_file_contents }}
+{{ src_file_contents | urlencode }} ##use jinja filter urlencode 
```

Comment:
I believe this is the opposite behavior of what we want.  

As per #213, the issue is that sometimes we are mistakenly urlencoding parts of the source code which we do not want to do.  I am not certain why/where the urlencoding is coming in, but the issue is that we are doing this somewhere in limited areas and we do not want to do that.

For example, instead of outputting: `public List<ShoppingCartItem>` 
we yield `public List&lt;ShoppingCartItem&gt;`


I mentioned in https://github.com/konveyor/kai/issues/213#issuecomment-2195111637 
I used `{{ src_file_contents | safe }}' 

Ideally we'd localize and understand why is it that only some parts of the files are being encoded, yet for short-term it could be sufficient to just do the ` | safe` that seems to address the problem

Reply:
Hello @jwmatthews! Apologies for the confusion. I misunderstood the specific encoding behavior we wanted to avoid, and I mistakenly added the urlencode filter, thinking we're not automatically escaping mentioned cases (<,>). 
Also, I'm not sure why the branch isn't reflected I'm merging from main :(
Please let me know if I shall raise another PR for this. Thanks!